custom_components/plugwise/config_flow.py

```diff
@@ -339,23 +339,23 @@ async def async_step_init(
 
         data = {
             vol.Optional(
-                timedelta(seconds=CONF_SCAN_INTERVAL),
+                timedelta(seconds=float(CONF_SCAN_INTERVAL)),
```

Comment:
Why add float here? Doesn't timedelta handle this internally, whether it's an int or a float?

Wait I remember Frenck saying that it should be a float, not an int: 2.5 secs :)

Reply:
It is a string, so we had to cast it somehow, and I already wondered about the float/int thing in #266

Reply:
```
class datetime.time(hour=0, minute=0, second=0, microsecond=0, [tzinfo](https://docs.python.org/3/library/datetime.html#datetime.tzinfo)=None, *, fold=0)
All arguments are optional. tzinfo may be None, or an instance of a tzinfo subclass. The remaining arguments must be integers in the following ranges:
...
0 <= second < 60,
...
```
So it must be int.

Reply:
Applied